bin/internal/release-candidate-branch.version

```diff
@@ -0,0 +1 @@
+flutter-3.14-candidate.15
```

Comment:
Is this part of the release process new?

Reply:
This has been the process for a while now.  It is built into conductor to update this file on the creation of a new beta.